src/Agent/NewRelic/Agent/Core/NewRelic.Agent.Core/Environment.cs

```diff
@@ -76,6 +76,8 @@ public Environment(ISystemInfo systemInfo, IProcessStatic processStatic, IConfig
                 if (!String.IsNullOrEmpty(configurationService.Configuration.AppSettingsConfigFilePath))
                     AddVariable("Application Config", () => configurationService.Configuration.AppSettingsConfigFilePath);
 
+                AddVariable("Server-Config Ignored", () => configurationService.Configuration.IgnoreServerSideConfiguration);
```

Comment:
Hi @browe
I'm sorry for the back and forth. I'm not sure whether this is the right place to report this config at this moment, so it would be great if you could remove this logic.

After you remove this logic, I will approve this PR and handle the rest.